.golangci.yaml

```diff
@@ -30,16 +30,23 @@ linters:
     # - nolintlint
     - revive
     - staticcheck
-    - structcheck
+    # - structcheck
     # - tagliatelle
     # - testpackage
     - typecheck
     - unconvert
     # - unparam
     - unused
-    - varcheck
+    # - varcheck
     - whitespace
 linters-settings:
+  depguard:
+    rules:
+      main:
+        list-mode: lax
+        deny:
+          - pkg: "github.com/sirupsen/logrus"
+            desc: not allowed
```

Comment:
How did you come up with these specific settings? Do we require everything as you added here? I could see some of these from depguard [docs](https://golangci-lint.run/usage/linters/#depguard).

Reply:
This linter has it merit, even if needed this a-bit-confusing configuration. I would prefer to keep it, even with this minor penalty.

Reply:
If so how about adding a comment alongside for `depguard` linter settings in _.golangci.yaml_ to make the intention clear?

Reply:
Fair enough. Done.